### PR TITLE
Disable disk metrics

### PR DESCRIPTION
**Pull request by krallin**
Disk metrics are pretty expensive for the Docker container handler,
since they amount to running `du` on the container's root directory,
which is somewhat costly in terms of CPU, especially when the host has
many containers. It can also be (very) costly in terms of disk when the
volume underlying the `/var/lib/docker` directory is slow(-ish), which
is our case since it's an EBS volume.

More importantly, those metrics are very useless:

- They can't be used to predict when `/var/lib/docker` might run out of
  space, since they only account for one container's usage (that's why
  they use `du`), whereas the underlying volume is used by multiple
  containers.
- They're not at all relevant to customers. Actual relevant disk usage
  is on DB data volumes, which is not captured by Cadvisor currently
  (and *we* wouldn't need to use `du` for those, sicne each DB has its
  own underlying volume).

Long story short, those metrics are costly and useless, so we might as
well get rid of them.

---

Note: TCP metrics are disabled by default, so this hasn't changed with
this patch.

### CODE REVIEW

**Comment by krallin** [comment]
cc @fancyremarker

**Comment by fancyremarker** [comment]
Thanks for explanation, LGTM!
